ansible/roles/java-base/vars/main.yml

```diff
@@ -14,7 +14,7 @@ packages: {
   'freebsd': 'openjdk8-jre',
   'smartos': 'openjdk8',
   'ubuntu': 'openjdk-8-jre-headless',
-  'ubuntu1404': 'oracle-java8-installer',
+  'ubuntu1404': 'openjdk-7-jre-headless',
```

Comment:
We need Java 8 on all of the jenkins nodes, regardless of the OS level as that is the minimum that works with our levels of Jenkins.

We had to manually update the PPC machines a while back, we should use that template to update these ansible scripts to get the right version on the PPC machines as well.

The templates are here: https://github.com/nodejs/build/issues/775#issuecomment-336245389

Reply:
If we wish to fetch the ppcle binaries from AdoptOpenJDK then we should consider using the api https://api.adoptopenjdk.net/README

Reply:
Fixed